src/components/metrics/card/index.js

```diff
@@ -158,9 +165,19 @@ class MetricCard extends Component {
 
     const titleView = !this.props.hovered && !isSelected && this._isTitle()
 
+    const relationshipClasses = {
+      INTERMEDIATE: 'intermediate',
```

Comment:
I don't think this style of constantitizing the class name actually adds much here; you still have the necessary magic string association between the function above and the map here. why not have the relationshipType function return relationshipClasses.INTERMEDIATE instead of magic strings?